tools/fleet-slackbot/claude-client.test.js

```
@@ -0,0 +1,127 @@
+
+  const result = await client.runAgentLoop("Tell me something.");
+
+  assert.equal(result, "Best-effort answer with partial data.");
+  assert.equal(
+    calls.length,
+    MAX_TOOL_ROUNDS + 1,
+    `expected ${MAX_TOOL_ROUNDS} tool rounds plus one forced no-tools call`
+  );
+
+  // All in-budget calls must NOT disable tool use.
+  for (let i = 0; i < MAX_TOOL_ROUNDS; i++) {
```

ESLint

[error] Unary operator '++' used.

(no-plusplus)